util/xsl/801.5.dyd.xsl

```diff
@@ -0,0 +1,125 @@
+<!--  This xsl:
+    * rename Dynawo.Electrical.Injectors.InjectorBG in Dynawo.Electrical.StaticVarCompensators.SVarCStandard
```

Comment:
Already done in others files no?

Reply:
Indeed it should not be there because it does not work if left in this file. I don't understand why but I have trouble using the text replace multiple times in the same file, if put separately it works but if put together it does not.. this is also why I have so much file.